test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/DynamicArchiveRelocationTest.java

```diff
@@ -91,8 +91,13 @@ static void doTest() throws Exception {
         String topArchiveName  = getNewArchiveName("top");
 
         String runtimeMsg = "Try to map archive(s) at an alternative address";
+        String runtimeRelocMsg = "runtime archive relocation start";
         String unmapPrefix = ".*Unmapping region #3 at base 0x.*";
         String unmapPattern = unmapPrefix + "(Bitmap)";
+        String archiveRelocPattern = ".*ArchiveRelocationMode == 1.*";
+        String unmapRgn1Pattern = ".*Unmapping region #1 at base 0x.*";
+        String unmapRgn0Prefix = ".*Unmapping region #0 at base 0x.*";
```

Comment:
unmapRgn0Prefix is only used to form unmapRgn0Pattern, they could merge together.